test/jest.config.integration.js

```diff
@@ -24,7 +24,7 @@ export default {
   ),
   setupFilesAfterEnv: [
     '<rootDir>/src/dev/jest/setup/after_env.integration.js',
-    '<rootDir>/plugins/opendistro_security/test/es/setup_es.js',
+    // '<rootDir>/plugins/opendistro_security/test/es/setup_es.js',
```

Comment:
Remove unused code

Reply:
actually those code are temporary commented out for fast testing, will uncomment them